modules/electrophysiology_browser/.gitignore

```diff
@@ -0,0 +1,2 @@
+js/electrophysiologySessionView.js
```

Comment:
don't we usually just include `js/*` in the module `.gitignore`? Are there other non-webpack generated files for the EEG browser?

Reply:
I was debating that... there are none no, but if one gets added later it will be automatically ignored. if we selectively ignore them however we dont run the risk of automatically ignoring new ones

Reply:
that beignsaid... I REALLY DONT CARE

Reply:
We shouldn't be putting anything directly in the `js` directory, there's only some modules that have an explicit file list there because they have old files that were grandfathered in and can't use the wildcard.

Reply:
@driusan @ridz1208  we could move those old files to the jsx directory as a solution.

Reply:
I don't think there are any old files in either of these modules.

Reply:
We should then just have `js/*` in the module like @driusan wrote.

Reply:
Can anyone make the changes directly on the pr ? I'm in meetings for the afternoon

Reply:
@ridz1208 added changes.

Reply:
@maltheism thanks